dataretrieval/nwis.py

```diff
@@ -73,34 +73,31 @@ def format_response(
         The formatted data frame
 
     """
-    mi = kwargs.pop('multi_index', True)
+    mi = kwargs.pop("multi_index", True)
 
-    if service == 'peaks':
+    if service == "peaks":
         df = preformat_peaks_response(df)
 
     if gpd is not None:
         if "dec_lat_va" in list(df):
-            geoms = gpd.points_from_xy(
-                df.dec_long_va.values,
-                df.dec_lat_va.values
-            )
+            geoms = gpd.points_from_xy(df.dec_long_va.values, df.dec_lat_va.values)
```

Comment:
This change seems a little weird, from multiple lines to one line, but not a biggie. I'm assuming this means that the line is less than 72 spaces.

Reply:
right, a little annoying but i'd live with this instance. Others might be worse such that we should try to address it now